File: cpp/bmi.cpp
/*
 * bmi.cpp
 */


#include <iostream>

using namespace std;

int main(int argc, char **argv)
{
	float wzrost = 0;
    float masa = 0;
    
    cout << "Podaj wzrost i masę ciała: " << endl;
    cin >> wzrost >> masa;
    
    if (masa/(wzrost*wzrost) < 18.5 )
    {
        cout << "niedowaga" << endl;
    }
    if  (masa/(wzrost*wzrost) < 24.9 && masa/(wzrost*wzrost) > 18.5)
    {
        cout << "norma" << endl;
    }
    if (masa/(wzrost*wzrost) >= 25 && masa/(wzrost*wzrost) < 30)
    {
        cout << "nadwaga" << endl;
    }
    if (masa/(wzrost*wzrost) >= 30)
    {
        cout << "otyłość" << endl;
    }
    
    
	return 0;
}

File: Python/bmi.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  bmi.py
#
#


def main(args):
    wzrost = float(input("Podaj wzrost: "))
    masa = float(input("Podaj masę ciała: "))
    if masa / (wzrost * wzrost) < 18.5:
        print("niedowaga")
    if masa / (wzrost * wzrost) < 24.9:
        print("norma")
    if masa / (wzrost * wzrost) < 25:
        print("nadwaga")
    if masa / (wzrost * wzrost) >= 30:
        print("otyłość")
    return 0


if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))
